sdk/src/test/scala/com/horizen/account/api/rpc/service/EthServiceTest.scala

```diff
@@ -189,6 +189,11 @@ class EthServiceTest extends JUnitSuite with MockitoSugar with ReceiptFixture wi
     )
   }
 
+  @Test
+  def eth_syncing(): Unit = {
+    assertEquals(false, ethService.execute(getRpcRequest()).asInstanceOf[Quantity].value)
```

Comment:
This will fail, as the function returns `Boolean` not `Quantity`.

Reply:
Right! UT fixed